### ISSUE TITLE
Make a full backup without a mailbox

### ISSUE THREAD

**Issue by juf00th** [lucascbeyeler/zmbackup#184]
How to make a full backup without a mailbox?

**Comment by lucascbeyeler** [comment]
Yes it's possible - update the README.md with examples on how to do this. But in short just run:

`$ zmbackup -f -ldp`
